docs/schema/constants/blend-mode.json

```diff
@@ -5,67 +5,67 @@
     "description": "",
     "oneOf": [
         {
-            "title": "Normal",
+            "title": "Blend Mode Normal",
```

Comment:
I definitely don't like repeating the enum name in its values to be honest

Reply:
I agree, it's not ideal, but then can you suggest a name? so we could avoid the generic names which may be created in other props

Reply:
What problem would it cause if "Normal" is used elsewhere too?

Reply:
answered in the comment below